src/controller/couchbase_lib/_bucket.py

```diff
@@ -150,18 +152,40 @@ def bucket_create(
             evictionpolicy=policy,
             bucket_type=bucket_type,
             bucket_compression=bucket_compression,
-            **env
+            **env,
         )
-        logger.debug("create bucket {}".format(command))
+        logger.debug(f"create bucket {command} for {bucket_name}")
         kwargs[ENV_VAR_KEY].update(env_vars)
         output, error, exit_code = utilities.execute_expect(
             self.connection, command, **kwargs
         )
         logger.debug(
-            "create bucket output: {} {} {}".format(output, error, exit_code)
+            f"create bucket output for {bucket_name}: "
+            f"{output} {error} {exit_code}"
         )
         helper_lib.sleepForSecond(2)
 
+        bucket_list = self.bucket_list()
+        if (
+            not any(item["name"] == bucket_name for item in bucket_list)
```

Comment:
in my opinion the statement 
`any(item["name"] == bucket_name for item in bucket_list)`
can be broken down into two steps for better readability:
1. create bucket name list using:
`bucket_name_list = [x["name"] for x in bucket_list]`

2. Modifying the condition to 
`bucket_name in bucket_name_list`

Reply:
Modified the code.